<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>지역 입력 :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_area">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>17</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(0, 170, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weather_btn">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>16</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>날씨조회</string>
    </property>
   </widget>
   <widget class="QLabel" name="area_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(4, 5, 63);
color: rgb(255, 15, 107);</string>
    </property>
    <property name="text">
     <string>날씨 조회 지역</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weather_image">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>날씨 이미지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temper_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼둥근헤드라인</family>
      <pointsize>31</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 89, 134);</string>
    </property>
    <property name="text">
     <string>10.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="yesterday_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>어제보다 1도 높아요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string>체감온도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);</string>
    </property>
    <property name="text">
     <string>미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 162);</string>
    </property>
    <property name="text">
     <string>초미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sensetemper_label">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>360</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid rgb(85, 255, 255)</string>
    </property>
    <property name="text">
     <string>12.1도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust01_info">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid rgb(121, 185, 37)</string>
    </property>
    <property name="text">
     <string>좋음</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust02_info">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid rgb(255, 255, 0)</string>
    </property>
    <property name="text">
     <string>나쁨</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
